modules/OFConnectionManager2/OFConnectionManager.mk

```diff
@@ -0,0 +1,14 @@
+
+###############################################################################
+#
+# Inclusive Makefile for the OFConnectionManager module.
+#
+# Autogenerated 2015-01-20 16:47:17.185057
+#
+###############################################################################
+OFConnectionManager_BASEDIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+include $(OFConnectionManager_BASEDIR)/module/make.mk
+include $(OFConnectionManager_BASEDIR)/module/auto/make.mk
+include $(OFConnectionManager_BASEDIR)/module/src/make.mk
+include $(OFConnectionManager_BASEDIR)/utest/_make.mk
+
```

Comment:
This file should not be checked in. There should be a .gitignore file to ignore it.